### DISCUSSION TITLE
Wordpress theme to upgrade from PHP 7 to PHP 8

### DISCUSSION THREAD

**Discussion by s-bh28** [rectorphp/rector#7578] [Q&A]
Hello, I have a Wordpress theme that is no longer maintained by its developers. Therefore, I need to upgrade its code myself. I came across this tool but not entirely sure how to use it. I downloaded PHPStorm but when I run the initial codes nothing happens? I was just wondering if there was a specific guide/similar scenario for WP themes that need to be upgraded to PHP 8 with rector.

**Comment by TheDesignerKhawaja** [comment]
Hello! So.... did you get any solution. I also need to upgrade a theme today from PHP7.3 to 8.x.

**Comment by s-bh28** [comment]
I had to switch to a new blank theme that was compatible with PHP8. My old
theme was not.
